staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6028,7 +6032,7 @@ type NodeStatus struct {
 	// +optional
 	Config *NodeConfigStatus `json:"config,omitempty" protobuf:"bytes,11,opt,name=config"`
 	// The available runtime handlers.
-	// +featureGate=RecursiveReadOnlyMounts
+	// +featureGate=RecursiveReadOnlyMounts,UserNamespacesSupport
```

Comment:
I think this should be on 2 lines

Reply:
@thockin Thanks for the suggestion. I will address this concern. I was looking for some docs here for the future reference. Any pointers?